interview-questions.md

```diff
@@ -7,53 +7,53 @@ Then, research the question to expand on your answer. Even if you feel you have
 
 1. Rails follows an MVC pattern. What does that mean?
 
-  Your answer:
+  Your answer: MVC stands for Model View Controller.
```

Comment:
This is a correct answer, but flex the elaboration muscles next week. try relating it back to work you've done or answering a few of the Big 6 Q's (WhoWhatWhenWhereWhyHow)